packages/shared/src/utils/config/config.js

```diff
@@ -91,3 +91,27 @@ export const getSocketURL = () => {
     }
     return server_url;
 };
+
+export const checkAndSetStagingEndpoint = () => {
+    if (/^staging\.deriv\.app$/i.test(location.hostname)) {
```

Comment:
Can we do this to allow `binary.sx` domains as well? It would be useful for test links as well.

Reply:
That's a good idea, I've added these changes.